packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -204,6 +205,29 @@ async function searchImages(value: string): Promise<string[]> {
   });
   return result;
 }
+
+let latestTagMessage: string | undefined = undefined;
+async function searchLatestTag(): Promise<void> {
+  if (imageNameIsInvalid || !imageToPull || imageToPull.includes(':')) {
+    latestTagMessage = undefined;
+    return;
+  }
+  try {
+    let image = imageToPull;
+    if (image.startsWith(DOCKER_PREFIX + '/')) {
+      image = image.slice(DOCKER_PREFIX.length + 1);
```

Comment:
question: should it be another constant ?

so no need to add a new / each time or to add +1 as well

Reply:
I'm not sure to find a constant name more explicit than these formulas